cpp/src/ray/test/cluster/test_python_call_cpp.py

```diff
@@ -0,0 +1,10 @@
+import ray
+import ray.cluster_utils
+
+
+def test_cross_language_cpp():
+    ray.init(
```

Comment:
Can we add tests for invalid cases (non xlang-serializable arguments)?

Reply:
I try to use bazel test, but there are many errors, however pytest is ok.
I'm not familiar with bazel test python in an independent directory, help wanted.

Reply:
Looks like you got bazel working here https://buildkite.com/ray-project/ray-builders-pr/builds/26009#7c8f6fae-4d88-4099-bafc-41b4446b3ec5/236-350 one more thing is to make sure to add a section at the end of `test_python_call_cpp.py` so the test will actually run (right now it's just noop)

```python
if __name__ == "__main__":
    import pytest
    import sys

    sys.exit(pytest.main(["-v", __file__]))
```

Reply:
Thank you, i will have a try.

Reply:
It works.